src/status_im2/config.cljs

```diff
@@ -38,7 +38,7 @@
 (def database-management-enabled? (enabled? (get-config :DATABASE_MANAGEMENT_ENABLED "0")))
 (def debug-webview? (enabled? (get-config :DEBUG_WEBVIEW "0")))
 (def collectibles-enabled? (enabled? (get-config :COLLECTIBLES_ENABLED "1")))
-(def test-stateofus? (enabled? (get-config :TEST_STATEOFUS "0")))
+(def test-stateofus? (enabled? (get-config :TEST_STATEOFUS "1")))
```

Comment:
probably this is leftover after development? it shouldn't be enabled by default

Reply:
I was wondering why we didn't enable it in development env, can i add `TEST_STATEOFUS=1` into `.env` so it enabled in dev env (i'm not sure) ?  @flexsurfer

Reply:
yes sure